base/applications/mspaint/precomp.h

```diff
@@ -21,7 +21,16 @@
 #include <shellapi.h>
 #include <htmlhelp.h>
 
-#include "definitions.h"
+#define WM_TOOLSMODELTOOLCHANGED WM_APP
+#define WM_TOOLSMODELSETTINGSCHANGED (WM_APP + 1)
+#define WM_TOOLSMODELZOOMCHANGED (WM_APP + 2)
+#define WM_PALETTEMODELCOLORCHANGED (WM_APP + 3)
+#define WM_PALETTEMODELPALETTECHANGED (WM_APP + 4)
+#define WM_IMAGEMODELDIMENSIONSCHANGED (WM_APP + 5)
+#define WM_IMAGEMODELIMAGECHANGED (WM_APP + 6)
+#define WM_SELECTIONMODELREFRESHNEEDED (WM_APP + 7)
```

Comment:
Please align/indent the values of these macros. (I know they weren't originally)

Reply:
OK.